src/TritonCTS/src/DbWrapper.cpp

```diff
@@ -125,7 +163,7 @@ void DbWrapper::initClock(odb::dbNet* net) {
         for (itr = iterms.begin(); itr != iterms.end(); ++itr) {
                 odb::dbITerm* iterm = *itr;
 
-                if (iterm->getIoType() != odb::dbIoType::INPUT) {
+                if (!(iterm->isInputSignal())) {
```

Comment:
This does not look right at all. A clock can be defined on an internal net and the driver is not an input iterm. In the problem this fix is trying to address the clock net that matters is driven by the pad instance.

Reply:
Here we iterate over all iterms in the net. What you're suggesting is that we should do the same for bterms? Or is there a way to know exactly which cells are connected to a certain net?